bundles/core/org.eclipse.smarthome.core.persistence/META-INF/MANIFEST.MF

```diff
@@ -7,13 +7,20 @@ Bundle-ManifestVersion: 2
 Bundle-License: http://www.eclipse.org/legal/epl-v10.html
 Bundle-SymbolicName: org.eclipse.smarthome.core.persistence
 Import-Package: org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.common.registry,
  org.eclipse.smarthome.core.items,
  org.eclipse.smarthome.core.library.types,
  org.eclipse.smarthome.core.persistence,
  org.eclipse.smarthome.core.types,
+ org.quartz,

```

Comment:
This is a problem. You might remember that we tried to get rid off Quartz in the core bundles (see https://github.com/eclipse/smarthome/issues/658), since it is not compatible with compact2 profile and also is pretty fat. So when overhauling the persistence APIs, I would definitely refrain from using this library.
I would hope that you can use @kgoderis' new scheduler implementation instead.